From 6b81707ce1ea49134ae0def3f05a2cf15b9626db Mon Sep 17 00:00:00 2001
From: Steven Hardy <shardy@redhat.com>
Date: Tue, 2 Jan 2018 17:49:12 +0000
Subject: [PATCH] Add missing user/project name in action context

These were removed in Ife653558bfcda794e7f37086832f70b0ad7c28a4
but that breaks any actions which require this data, such as
some TripleO actions.

Closes-Bug: 1740891
Change-Id: I777b1f7c7012b735805e8585938b5ce5dec31d26
---
 mistral/context.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mistral/context.py b/mistral/context.py
index 4887f15f..0c346450 100644
--- a/mistral/context.py
+++ b/mistral/context.py
@@ -270,6 +270,8 @@ def create_action_context(execution_ctx):
         is_target=context.is_target,
         is_trust_scoped=context.is_trust_scoped,
         project_id=context.project_id,
+        project_name=context.project_name,
+        user_name=context.user_name,
         redelivered=context.redelivered,
         region_name=context.region_name,
         service_catalog=context.service_catalog,
-- 
2.14.3

